beluga_amcl/cmake/ament.cmake

```diff
@@ -66,17 +54,15 @@ rclcpp_components_register_node(
 
 ament_export_dependencies(
   beluga
+  beluga_ros
   bondcpp
   nav2_msgs
   rclcpp
   rclcpp_components
   rclcpp_lifecycle
-  sensor_msgs
-  tf2
-  tf2_eigen
-  tf2_geometry_msgs)
+  std_srvs)
 ament_export_include_directories("include/${PROJECT_NAME}")
-ament_export_definitions(BELUGA_AMCL_ROS_VERSION=2)
+ament_export_definitions(BELUGA_ROS_VERSION=2)
```

Comment:
@nahueespinosa meta: should `beluga_amcl` re-export definitions already exported by `beluga_ros`? If we need to condition compilation in `beluga_amcl` based on the ROS distribution version, in addition to that in `beluga_ros` already (it doesn't look like it), shouldn't the definition remain `BELUGA_AMCL_ROS_VERSION`? Same for ROS 1.

Reply:
Ah, thanks for catching that! Fixed in https://github.com/Ekumen-OS/beluga/pull/270/commits/f209efc8852159635c04fd0b6269ff4e75f4627e.